# Request message types
float32 x
float32 y
float32 z
float32 quat_x
float32 quat_y
float32 quat_z
float32 quat_w
---
# Response message types
string success